frontend/src/logic/providers/schedule-provider.model.ts

```diff
@@ -0,0 +1,57 @@
+import { WorkerType } from "../../common-models/worker-info.model";
+import { ScheduleDataModel } from "../../common-models/schedule-data.model";
+import { ShiftsProvider } from "./shifts-provider.model";
+import { MetadataProvider } from "./metadata-provider.model";
+import { ChildrenInfoProvider } from "./children-info-provider.model";
+import { ExtraWorkersInfoProvider } from "./extra-workers-info-provider.model";
+
+export interface Sections {
```

Comment:
Jeżeli będziemy tworzyć nową sekcję, to potrzeba będzie dodać w to miejsce pole z nazwą tej sekcji i abstrakcyjnym providerem, który odpowiada tej sekcji

Reply:
Moim zdaniem, to jest sensowne rozwiązanie, bo mamy jedno źródło prawdy, które definiuje jakie sekcje istnieją. Powiedźcie co wy o tym sądzicie ?

Reply:
Bardzo mi się to podoba :D